### DISCUSSION TITLE
Is management interface also monitored?

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2445] [Unsupported Versions] [answered]
This may be a stupid question, but is the Management interface also Monitored?  During the setup the Management interface is not listed when asked what interfaces to define as Monitored.  In passed years I could always choose the management interface as also a monitored interface.  But with v2 I'm not seeing any traffic being collected via the management interface.  Please advise.

**Comment by dougburks** [comment, marked as answer]
No, the management interface is not monitored the way that monitored (sniffing) interfaces are.
